packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1699,8 +1700,24 @@ export class ContainerRuntime
 			this.remoteMessageProcessor.clearPartialMessagesFor(clientId);
 		});
 
-		// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-		this._audience = audience!;
+		this._audience = audience;
+		if (!Object.prototype.hasOwnProperty.call(audience, "currentClientId")) {
```

Comment:
I'm curious if this is a pattern we use anywhere else for back-compat

Reply:
I doubt that. That was the fastest thing I came up with. I can wrap it with another object, probably it's a bit future proof, but it's a bunch of plumbing I'm not looking for :)

Reply:
FWIW, Object.hasOwn() is the recommended replacement for hasOwnProperty()

Reply:
```suggestion
		if (!Object.hasOwn(audience, "currentClientId")) {
```

Reply:
It errors out:
 Property 'hasOwn' does not exist on type 'ObjectConstructor'. Do you need to change your target library? Try changing the 'lib' compiler option to 'es2022' or later

Reply:
Oh sad - maybe it's time to update to 2022 :)

Reply:
I had to change it to "in" test - this test was not testing it property (was returning false for getter)